Fix windows release directory renaming, regex string escaping and indentation

oAGoulart opened the pull request:
**1**. Fixed bug caused by the use of `%` on replacement string when calling `string.gsub` (**lua**>=5.2). According to lua-users: ([**link**](http://lua-users.org/wiki/LuaFiveTwo))
> ### string.gsub repl
> string.gsub [[6]](http://www.lua.org/manual/5.2/manual.html#pdf-string.gsub) now raises an error if the replacement string contains a '%' followed by a character other than the permitted '%' or digit.

In short, the replacement string does not require escaping of *magic characters*.

**2**. Function `utils.lua.escape_string_regex` recreated with single `string.gsub` call.

**3**. Converted a few lines of code with three spaces of indentation to two.

MisterDA commented:
Ideally, I’d prefer to have one commit fixing the bug, and one commit fixing the indentation.
You can also squash the commit fixing the typo (but I can do that during the merge). Don’t hesitate to force-push.
Can you do that?

MisterDA committed:
Fix indentation.

oAGoulart committed:
Fix windows release directory renaming.

Fixed bug caused by the use of `%` on replacement string when calling
`string.gsub` (lua >= 5.2). According to
[lua-users](http://lua-users.org/wiki/LuaFiveTwo):

> ### string.gsub repl
>
> string.gsub
> [[6](http://www.lua.org/manual/5.2/manual.html#pdf-string.gsub)] now
> raises an error if the replacement string contains a '%' followed by
> a character other than the permitted '%' or digit.

In short, the replacement string does not require escaping of *magic
characters*.

oAGoulart committed:
utils.lua.escape_string_regex rewritten with single string.gsub call.

MisterDA commented:
Took the liberty of doing the work myself and force-pushing. Thanks for the patches!
